docs/references.md

```diff
@@ -0,0 +1,15 @@
+#References
+
+List of references used in this documentation:
+
+1. <a name="nyaml GitHub repository"></a>[nyaml GitHub repository](https://github.com/FAIRmat-NFDI/nyaml)
+2. <a name="NeXus-FAIRmat"></a>[NeXus-FAIRmat](https://fairmat-experimental.github.io/nexus-fairmat-proposal/)
+3. <a name="FAIRmat-NeXus GitHub repository"></a>[FAIRmat-NeXus GitHub repository](https://github.com/FAIRmat-NFDI/nexus_definitions)
+4. <a name="official site"></a>[Official NeXus website](https://manual.nexusformat.org/index.html)
+5. <a name="j_appl_cryst_2015"></a>[J. Appl. Cryst. (2015). 48, 301-305](https://doi.org/10.1107/S1600576714027575)
+6. <a name="XML"></a>[XML](https://www.w3.org/TR/REC-xml/REC-xml-20081126.xml)
+7. <a name="nyaml"></a>[nyaml](https://github.com/FAIRmat-NFDI/nyaml/tree/main)
+8. <a name="NeXus official website"></a>[NeXus official website](https://www.nexusformat.org/)
+9. <a name="YAML"></a>[YAML](https://yaml.org/)
+10. <a name="lxml"></a>[lxml](https://lxml.de/)
```

Comment:
Follow these rules:
http://citebay.com/how-to-cite/lxml/

Reply:
This is not a citation as goes with publication. But a general reference link used software or information. Or? But the such citation must come in Joss paper as you mentioned in here.